dropshot/Cargo.toml

```diff
@@ -11,24 +11,28 @@ keywords = ["rest", "openapi", "async"]
 categories = ["network-programming", "web-programming::http-server"]
 
 [dependencies]
+async-stream = "0.3.2"
 async-trait = "0.1.24"
 base64 = "0.13.0"
 bytes = "1"
 futures = "0.3.1"
+futures-core = "0.3.16"
 hostname = "0.3.0"
 http = "0.2.0"
 indexmap = "1.0.0"
 openapiv3 = "0.5.0"
 paste = "1.0.0"
 percent-encoding = "2.1.0"
 proc-macro2 = "1.0.27"
+rustls = "0.19.1"
```

Comment:
Dunno how much this is on your radar, but it looks like a 0.20 beta exists, and changes some parts of the TLS api.

I saw an example here: https://github.com/ctz/rustls/blob/main/rustls-mio/examples/tlsserver.rs ; 

The [rustls::ServerConnection object](https://docs.rs/rustls/0.20.0-beta2/rustls/struct.ServerConnection.html) looked new.

Reply:
Upgraded